src/libYARP_manager/src/manager.cpp

```diff
@@ -1021,7 +1021,9 @@ bool Manager::connect()
     //connector.init();
     CnnIterator cnn;
     for(cnn=connections.begin(); cnn!=connections.end(); cnn++) {
-        if( !connector.connect((*cnn).from(), (*cnn).to(),
+        if( !(*cnn).getFromExists() ||
+            !(*cnn).getFromExists() ||
```

Comment:
Should that be `getToExists()`?

Reply:
Good catch! Copy paste error :sweat_smile: